tests/Integration/Integration.Tests/PurchaseOrdersReportModuleTests/WhenGettingOrdersBySupplier.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Linn.Purchasing.Integration.Tests.PurchaseOrderReportModuleTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Web;
+
+    using FluentAssertions;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Reporting.Resources.ReportResultResources;
+    using Linn.Purchasing.Integration.Tests.Extensions;
+    using Linn.Purchasing.Resources;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenGettingOrdersBySupplier : ContextBase
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            var reportReturnResource = new ReportReturnResource();
+            var reportResult = new ReportResultResource
+                                   {
+                                       displaySequence = 1,
+                                       title = new DisplayResource("potat"),
+                                       results = new List<ResultDetailsResource>
+                                                     {
+                                                         new ResultDetailsResource
+                                                             {
+                                                                 rowTitle = new DisplayResource("rowtitle"),
+                                                                 rowType = "string",
+                                                                 values = new List<ValueResource>
+                                                                              {
+                                                                                  new ValueResource(1)
+                                                                                      {
+                                                                                          textDisplayValue =
+                                                                                              "ramen noodles"
+                                                                                      }
+                                                                              }
+                                                             }
+                                                     }
+                                   };
+
+            reportReturnResource.ReportResults.Add(reportResult);
+
+            this.FacadeService
+                .GetOrdersBySupplierReport(Arg.Any<OrdersBySupplierSearchResource>(), Arg.Any<IEnumerable<string>>())
+                .Returns(new SuccessResult<ReportReturnResource>(reportReturnResource));
+            var resource = new OrdersBySupplierSearchResource { From = "2/11/21", To = "2/12/21" };
+
+            var builder = new UriBuilder("http://localhost:51699/purchasing/reports/orders-by-supplier/report");
```

Comment:
I think I managed to get query params in here in a (slightly) neater way if you look at some of my SupplierModuleTests